data/src/test/kotlin/com/vultisig/wallet/data/usecases/AesEncryptionTest.kt

```
@@ -67,4 +77,149 @@ class AesEncryptionTest {
+                (0 until 32).map { taskIndex ->
+                    pool.submit {
+                        try {
+                            repeat(100) { iteration ->
+                                val payload = "t$taskIndex-i$iteration".toByteArray(Charsets.UTF_8)
+                                val encrypted = aes.encrypt(payload, pw)
+                                val decrypted = aes.decrypt(encrypted, pw)
+                                if (decrypted == null || !decrypted.contentEquals(payload)) {
+                                    failures.incrementAndGet()
+                                }
+                            }
+                        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)